CP/linkedlist/mergesort.cpp

```diff
@@ -0,0 +1,163 @@
+// In this code script we have done "sorting of linked list using mergesort"
```

Comment:
Hi @amanvatss , Thanks for contributing and being a part of Hacktoberfest 2024. 
Two small requests-
1. Please move this to https://github.com/Kavya-24/Snippets/tree/master/Algorithms/Sorting%20Algorithms
2. Please remove the executable binary from the commit. 


Thanks !